### DISCUSSION TITLE
slurky cussions!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by fr4nk014** [fr4nk014/slurky#2] [General]
talki

**Comment by zzamizz** [comment]
weather looks fire tomorrow

**Reply by fr4nk014** [reply]
yes

**Comment by fr4nk014** [comment]
minkäs värinen

**Reply by zzamizz** [reply]
on se

**Reply by fr4nk014** [reply]
minkäs tyyppiset pizzat hotkaistaan sitten ensviikolla?

**Reply by zzamizz** [reply]
1. MARGARETA

2. BOLOGNESE

3. OPERA SPECIAL

4. BELLA

5. LAGUNA

6. AMERICANO

7. FRUTTI DI MARE

8. ROMANO

9. VEGETARIANA

10. MEXICANA

11. KEBAB PIZZA

12. BLONDIN VEGE

13. SALLYN VEGE

14. PARADISO

15. DILLINGER

16. QUATTRO

17. BARBEQUE PARTY

18. TOSI MIEHEN VEGE

19. SMETANA SPECIAL

20. SKY EXPRESS

21. OMA VALINTA

**Reply by fr4nk014** [reply]
> > - ## MISSÄ JÄRJESTYKSESSÄ~~???~~??

**Reply by zzamizz** [reply]
![20220810_021346.jpg](https://user-images.githubusercontent.com/37944755/185024062-2f01cc42-b48c-486b-8dac-45a3a9fb5e1b.jpg)
